### DISCUSSION TITLE
Add auto dark mode on Typst web app

### DISCUSSION THREAD

**Discussion by mendax1234** [typst/typst#2516] [Ideas]
Does anyone now whether I can change between the light and dark theme according to the system in typst web app. I think this is an important function.

**Comment by Enter-tainer** [comment]
can you elaborate your use case? for printing, a "dark mode" pdf mostly doesnt make sense.

**Reply by laurmaedje** [reply]
I think they are talking about the web app.

**Reply by mendax1234** [reply]
Yeah, it should be the web app.
